platforms/core-runtime/launcher/src/main/java/org/gradle/tooling/internal/provider/LauncherServices.java

```diff
@@ -227,7 +227,7 @@ BuildTreeActionExecutor createActionExecutor(
             FileHasherStatistics.Collector fileHasherStatisticsCollector,
             DirectorySnapshotterStatistics.Collector directorySnapshotterStatisticsCollector,
             BuildOperationRunner buildOperationRunner,
-            BuildLayout buildLayout,
+            BuildLocations buildLayout,
```

Comment:
💅 I agree with not necessarily renaming other symbols but I'd rename service factory parameters such as this one since there's a clear pattern to follow.